Source/UserManagement/Domain/DataCollectors/CommandHandlers/IDataCollectorCommandHandler.cs

```diff
@@ -0,0 +1,12 @@
+using doLittle.Runtime.Commands;
+using Domain.DataCollectors.Commands;
+
+namespace Domain.DataCollectors.CommandHandlers
+{
+    public interface IDataCollectorCommandHandler : ICanHandleCommands
```

Comment:
I don't think there is any benefit in adding an interface for IDataCollectorCommandHandler.

I might group Handle methods in a particular class for convenience (where they are related to the same Aggregate Root and have the same dependencies) but since the infrastructure / framework has the responsibility for hooking up commands to the handlers, this is not something I would test / mock out.